src/components/Card/Card.tsx

```diff
@@ -12,6 +13,7 @@ interface CardProps {
   imageAlt: string;
 }
 const Card: React.FC<CardProps> = ({
+  customStyle,
```

Comment:
ふつうに style でいいと思う

Reply:
importしているstyleと被ってしまうことを避けるために変えてみました。
ここはかぶっていても大丈夫ですか？

Reply:
あーなるほど...
それなら props を style にして、 Card 引数側で
```suggestion
  style: customStyle,
```
にすれば親で style として渡してもこっちでは customStyle として受け取れます

Reply:
ありがとうございます